Channels/Integration/01e_Magento2.md

```diff
@@ -0,0 +1,167 @@
+        - *Suspect Fraud*
+        - *Processing*
+        - *Pending Payment*
+        - *Payment Review*
+        - *Pending*
+        - *On Hold*
+        - *Complete*
+        - *Closed*
+        - *Canceled*
+        - *PayPal Canceled Reversal*
+        - *Pending PayPal*
+        - *PayPal Reversed*
+        - *Refunded*
+
+6. Configure the following settings to define the desired order statuses or states to be set after a certain event has occurred in the *Update order state and status on certain events* box:   
+
+    - Click the *Order state to be set in shop after successful import* drop-down list and select the desired state. The following states are available:  
+        - **new**
+        - **pending**
+        - **pending_payment**
+        - **payment_review**
+        - **processing**
+        - **holded**
+        - **completed**
+        - **closed**
+        - **canceled**  
+
+    - Enter the desired order status to be set when an order has been successfully imported in the *Order status to be set in shop after successful import* field.
+    
+    - Enter the desired order status to be set when an order has been partially shipped in the *Order status to be set in shop after partially shipped* field.
+
+    - Click the *Order state to be set in shop after fully shipped* drop-down list and select the desired state. The following states are available:  
+        - **new**
+        - **pending**
+        - **pending_payment**
+        - **payment_review**
+        - **processing**
+        - **holded**
+        - **completed**
+        - **closed**
+        - **canceled**  
+
+    - Enter the desired order status to be set when an order has been fully shipped in the *Order status to be set in shop after fully shipped* field.
+
+    - Enter the desired order status to be set when an order has been partially paid in the *Order status to be set in shop after partially paid* field.
+
+    - Enter the desired order status to be set when an order has been fully paid in the *Order status to be set in shop after fully paid* field.
+
+    - Enter the desired order status to be set when a tracking number has been set for the order in the *Order status to be set in shop after tracking number set* field.
+
+7.  If necessary, configure the following settings in the *Magento invoice handling* box:
+
+    - Enable the *Create an invoice in Magento for orders that are fully paid* toggle. 
+    
+        > [Info] If enabled, invoice creation is triggered in Magento 2 as soon as an order is fully paid. This function is particularly relevant for the correct processing of vouchers.  
+    
+        The *Notify customer about the invoice* toggle is unlocked.
+
+    - Enable the *Notify customer about the invoice* toggle to notify customers when an invoiced is generated.
```

Comment:
```suggestion
7.  If necessary, configure the following settings in the *Magento invoice handling* section:

    + Enable the *Create an invoice in Magento for orders that are fully paid* toggle to create invoices in Magento 2 for fully paid orders and ensure the correct processing of vouchers.  
        
    + Enable the *Notify customer about the invoice* toggle to notify customers when an invoiced is generated. This toggle is only unlocked if the *Create an invoice in Magento for orders that are fully paid* toggle is enabled.
```
Ich muss nochmal checken, ob ich bislang in Unterpunkten auch results eingeschlossen habe. Ich meine, das habe ich bislang immer nur am Ende eines steps gemacht.